### PR TITLE
Added editor to table designer

### PR DESCRIPTION
**Pull request by abist**
Screen Shot 2021-11-02 at 5 27 01 PM

### CODE REVIEW

File: extensions/mssql/package.json [hunk 1]
```diff
@@ -448,6 +448,16 @@
         }
       ],
       "objectExplorer/item/context": [
+        {
```
Comment (alanrenmsft) [inline extensions/mssql/package.json:451]: should not be included

File: src/sql/workbench/contrib/tableDesigner/browser/tableDesignerTextEditor.ts [hunk 2]
```diff
@@ -4,8 +4,15 @@
...
+import { Dimension } from 'vs/base/browser/dom';
 import { Event, Emitter } from 'vs/base/common/event';
+import { Schemas } from 'vs/base/common/network';
+import { URI } from 'vs/base/common/uri';
+import { ITextModel } from 'vs/editor/common/model';
 import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
+import { UntitledTextEditorInput } from 'vs/workbench/services/untitled/common/untitledTextEditorInput';
+import { UntitledTextEditorModel } from 'vs/workbench/services/untitled/common/untitledTextEditorModel';
 
 // TODO: Implement the text editor
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/tableDesigner/browser/tableDesignerTextEditor.ts:17]: remove the todo

File: src/sql/workbench/contrib/tableDesigner/browser/tableDesignerTextEditor.ts [hunk 3]
```diff
@@ -14,7 +21,27 @@ export class TableDesignerTextEditor implements DesignerTextEditor {
...
+	constructor(
+		private _container: HTMLElement,
+		@IInstantiationService private _instantiationService: IInstantiationService) {
+		this._editor = this._instantiationService.createInstance(TableDesignerBaseTextEditor);
+		this._editor.create(this._container);
+		this._editor.setVisible(true);
+		this._untitledTextEditorModel = this._instantiationService.createInstance(UntitledTextEditorModel, URI.from({ scheme: Schemas.untitled }), false, undefined, 'sql', undefined);
+		this._editorInput = this._instantiationService.createInstance(UntitledTextEditorInput, this._untitledTextEditorModel);
+		this._editor.setInput(this._editorInput, undefined, undefined);
+		this._editorInput.resolve().then((model) => {
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/tableDesigner/browser/tableDesignerTextEditor.ts:38]: how do I set the content?

Comment (abist) [reply src/sql/workbench/contrib/tableDesigner/browser/tableDesignerTextEditor.ts:40]: in the set content function, updated now

**Review by alanrenmsft — APPROVED** [review]

**Comment by alanrenmsft** [comment]
image

**Pull request merged**